### ISSUE TITLE
Slideshow progresses even when image has not loaded yet

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud-vue#2123]
(Sort of related to https://github.com/nextcloud/nextcloud-vue/pull/2122 @szaimen )

Currently when you play a slideshow, and the image (or its preview) hasn’t loaded, the slideshow timer still runs. So maybe you don’t even get to see the image and it jumps to the next one.

→ The slideshow timer should be paused while the image/preview is still loading.

**Comment by szaimen** [comment]
Yes, would like to fix this, too but is unfortunately not a simple one-liner ;)
Also see https://github.com/nextcloud/viewer/issues/39
